provider/cmd/pulumi-gen-aws-native/supported-types.txt

```diff
@@ -133,7 +135,6 @@ AWS::EFS::AccessPoint
 AWS::EFS::FileSystem
 AWS::EFS::MountTarget
 AWS::EKS::Addon
-AWS::EKS::Cluster
```

Comment:
Are we cool with this going away?

Reply:
They apparently rolled this back due to some bug, so I don't think it's working now anyway.

Reply:
Oh. So is https://github.com/pulumi/examples/tree/master/aws-native-ts-ecs broken?

Reply:
No, this is EKS, not ECS.

Reply:
Yes - I realized that just now 🤦🏾.